src/main/java/eiteam/esteemedinnovation/commons/init/ContentModule.java

```diff
@@ -27,20 +27,20 @@
  * <br />
```

Comment:
Formatting in this class is not consistent with the code style. Notably:
* [The newlines after { for empty blocks](https://github.com/Esteemed-Innovation/Esteemed-Innovation/blob/1.12/STYLE_GUIDE.md#empty-blocks)

And things which are not explicitly defined in the code style document but are the style elsewhere in the code:
* Tabs before `<li>`s
* No newlines before `@param`s
* No unneeded whitespace after `@param`s